src/System.Linq.Expressions/tests/Unary/UnaryIsFalseNullableTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Xunit;
+
+namespace Tests.ExpressionCompiler.Unary
+{
+    public static class UnaryIsFalseNullableTests
+    {
+        #region Test methods
+
+        [Fact] //[WorkItem(3196, "https://github.com/dotnet/corefx/issues/3196")]
```

Comment:
Roslyn uses WorkItemAttribute as a standardized way for regression tests to refer to bugs. It has been found very useful in the past, since bug descriptions and discussions can contain a lot of relevant information, that would be hard to make a part of a test. 
Being an attribute makes it easy to find in an automated way if so needed (bugs migrated to alternative store or something)

Turns out Linq.Expression does not have any convention for regression tests. I suggested commented out attribute as a temporary measure to not lose the information about corresponding bug while not blocking the change on an infrastructure decision.

The plan is to get an agreement on these kind of adornments (most likely some kind of attribute) and search/replace the comments as a separate change. 

It seems better than to rely on natural text comments or not preserving this data at all.